hack/test/local_test.sh

```diff
@@ -66,23 +66,24 @@ if [[ ! $(pwd) =~ .*certman-operator$ ]]; then
   exit
 fi
 
-echo "Checking if docker service is active"
-systemctl is-active docker
+#echo "Checking if docker service is active"
+#systemctl is-active docker
 
 testdir="${initial_dir}/hack/test"
 tmpdir="${initial_dir}/hack/test/tmp"
 mkdir ${tmpdir}
 
 # Start minikube with extra memory (needed for the go build)
-minikube start -p certtest --memory='2500mb'
+minikube start -p certtest --memory=6g --bootstrapper=kubeadm --extra-config=kubelet.authentication-token-webhook=true --extra-config=kubelet.authorization-mode=Webhook --extra-config=scheduler.address=0.0.0.0 --extra-config=controller-manager.address=0.0.0.0 --extra-config=apiserver.service-node-port-range=1-65535
 kubectl config use-context certtest
 
 # Install openshift router
 cd $tmpdir
 git clone git@github.com:openshift/router.git
 cd router
-kubectl create ns openshift-ingress
-kubectl create -n openshift-ingress -f deploy/
+kubectl create -n openshift-ingress -f deploy/route_crd.yaml
```

Comment:
Can we move this into a hack/test/deploy directory?

Reply:
that specific resource is from the openshift/router git repo, so it's not something we have to worry about